mobile/src/main/res/layout/merge_item.xml

```diff
@@ -2,60 +2,57 @@
 <merge xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     tools:background="@color/white"
+    tools:baselineAligned="false"
     tools:layout_height="wrap_content"
     tools:layout_width="match_parent"
-    tools:paddingLeft="8dp"
-    tools:paddingRight="0dp"
     tools:parentTag="android.widget.LinearLayout">
 
-
     <ImageView
         android:id="@+id/item_drag_handle"
         style="@style/Chunks.Item.Icon.DragHandle"
-        android:layout_width="48dp"
-        android:layout_height="48dp"
-        android:layout_gravity="center_vertical"
+        android:layout_width="wrap_content"
+        android:layout_height="48sp"
+        android:paddingStart="20sp"
+        android:paddingEnd="12sp"
         android:src="@drawable/chunks_item_drag_handle" />
 
     <CheckBox
         android:id="@+id/item_check_box"
         style="@style/Chunks.Item.CheckBox"
-        android:layout_width="48dp"
-        android:layout_height="48dp"
-        android:layout_gravity="center_vertical" />
+        android:layout_width="wrap_content"
+        android:layout_height="48sp" />
 
     <TextView
         android:id="@+id/item_text_view"
         style="@style/Chunks.Item.Text"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_gravity="center_vertical"
+        android:layout_marginStart="8sp"
+        android:layout_marginTop="13sp"
```

Comment:
since the linearlayout isn't baseline aligned anymore, we need to manually push the textview down so the first line aligns with its siblings.

using sp across the board means that if the textsize is changed, it stays aligned.